### ISSUE TITLE
Every reCaptcha attempt fails

### ISSUE THREAD

**Issue by RichmondCoB** [OddOneOut/Better-WordPress-reCAPTCHA#28]
My website http://www.richmondcob.org/ has used reCaptcha since we switched to WordPress a few years ago, but this year reCaptcha has started to fail constantly, as in neither I nor any site visitors can commit any post that uses reCaptcha. What's going on here? I don't want to go back to the land of spam comments, but this is unacceptable.

**Comment by kminh** [comment]
Does it work for you here: https://www.google.com/recaptcha/api2/demo ?

**Comment by RichmondCoB** [comment]
Yes, that works fine. Feel free to visit my website to try out the
reCaptcha there on our contact page
http://www.richmondcob.org/about/contact-us/

On Tue, May 31, 2016 at 10:47 AM, kminh notifications@github.com wrote:

> Does it work for you here: https://www.google.com/recaptcha/api2/demo ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/OddOneOut/bwp-recaptcha/issues/28#issuecomment-222711821,
> or mute the thread
> https://github.com/notifications/unsubscribe/ASwfYTYHxLcn9qTDsRUuqjzWVQ6skMXuks5qHEn_gaJpZM4IqkZW
> .

## 

Grace and peace,

Joseph Tolton
Office Manager
Richmond Church of the Brethren
